src/types.ts

```diff
@@ -42,13 +42,24 @@ export interface CopilotFacade {
      */
     perform: (...steps: string[]) => Promise<string>;
 
+
+     /**
+     * Preforms actions untill it reaches the goal.
+     * @example Book a class
+     * @example Buy a product
+     * @example win the shape matching game
```

Comment:
the `@example` is not clear enough for users that aren't familiar with our cool game demo
let's change it to something like `login with user "testuser" and password "testpassword123"`. It will be clearer to what scenarios we aim.